src/__tests__/loginFlow.integration.test.js

```diff
@@ -0,0 +1,74 @@
+require('dotenv').config()
+import { mount } from '@vue/test-utils'
+import App from '../App.vue'
+import wait from 'waait'
+import MockHposApi from '../../mock-hpos-api'
+import router from '../router'
+
+describe('Login Flow', () => {
+  const email = "test@test.com"
+  const password = "passw0rd"
+
+  let mockHposApi
+
+  beforeAll(async () => {
+    mockHposApi = await MockHposApi.start(Number(process.env.VUE_APP_HPOS_PORT) + 1, email, password)
+  })  
+
+  afterAll(() => {
+    mockHposApi.close()
+  })
+
+  beforeEach(async () => {
+    router.push('/')
+    await router.isReady()
+  })
+
+  it('prevents login with incorrect credentials', async () => {
+    const wrongEmail = 'wrong@email.com'
+    const wrongPassword ='asdfgh'
+
+    const wrapper = mount(App, {
+      global: {
+        plugins: [router]
+      }
+    })
```

Comment:
Should we also add a test that the right email/password with wrong pubkey also fails?

Reply:
I can add it if you feel strongly about it. It would require a little bit of changing how the code works to make that testable.

Reply:
Nevermind then :)